apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -654,6 +657,53 @@ type RateLimitServiceConfig struct {
 	EnableResourceExhaustedCode *bool `json:"enableResourceExhaustedCode,omitempty"`
 }
 
+// TracingConfig defines properties for exporting trace data to OpenTelemetry.
+type TracingConfig struct {
+	// IncludePodDetail defines a flag.
+	// If it is true, contour will add the pod name and namespace to the span of the trace.
+	// the default is true.
+	// +optional
+	IncludePodDetail *bool `json:"IncludePodDetail,omitempty"`
```

Comment:
Is this field name starting with a capital letter intentionally? Let's use lowercase to make it consistent with the rest. If you decide to leave it uppercase first letter, then the below would be equivalent:
```go
	IncludePodDetail *bool `json:",omitempty"`
```

Reply:
This is a mistake, it should start with a lowercase letter